pkg/operation/botanist/clusteridentity_test.go

```diff
@@ -127,4 +143,21 @@ var _ = Describe("ClusterIdentity", func() {
 			shoot.Status.ClusterIdentity = pointer.StringPtr(expectedShootClusterIdentity)
 		}),
 	)
+
+	Describe("#DeployClusterIdentity()", func() {
```

Comment:
nit, for consistency
```suggestion
	Describe("#DeployClusterIdentity", func() {
```